secator/ai/history.py

```
@@ -21,17 +21,21 @@ def get_context_window(model: str) -> int:
+    from secator.config import CONFIG
     import litellm
     try:
         info = litellm.get_model_info(model)
-        context_window = info.get("max_input_tokens") or info.get("max_tokens", 128_000)
+        if not info:
+            debug('context_window=config fallback (empty model info)', sub='runner.ai.context', obj={'model': model})
+            return CONFIG.addons.ai.context_window
+        context_window = info.get("max_input_tokens") or info.get("max_tokens") or CONFIG.addons.ai.context_window
         debug(f'context_window={context_window}', sub='runner.ai.context', obj={'model': model})
         return context_window
     except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)